src/components/OurWork.astro

```diff
@@ -1,209 +1,201 @@
 ---
 import CtaButtom from './CtaButton.astro'
 const data = Astro.fetchContent('../pages/work/*.md')
-
 ---
-<h2 id="work-title"><a class="link" href="/work">Our Work</a></h2>
-<div id="button-continer">
```

Comment:
I cut out this `#button-container`. It wasn't really doing anything (let alone containing a button). Also got rid of most of the IDs here. Since the style is scoped, I think it reads better to just select the element. And in the event we ever end up loading two on a page, we don't want the two IDs conflicting.